backend/clap/src/main/java/zpi/ppea/clap/controller/StudyVisitsController.java

```diff
@@ -0,0 +1,30 @@
+package zpi.ppea.clap.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import zpi.ppea.clap.config.ValueConfig;
+import zpi.ppea.clap.dtos.StudyVisitDto;
+import zpi.ppea.clap.security.FirebaseAgent;
+import zpi.ppea.clap.service.StudyVisitService;
+
+@RestController
+@RequestMapping("/visits")
```

Comment:
To discuss - shouldn't we keep the visit endpoints within the Submission Controller? In case we would have to add Update/Create operations for the visits, the path should be `/visits/{visitId}` which will collide with the `/visits/{submissionId}`. Well, it won't collide as would probably a different HTTP method, but having different path parameters for the same path but a different method doesn't sound good to me.

Reply:
The path I'd go for can be `/submissions/${submissionId}/visits`